modules/post/windows/gather/credentials/securecrt.rb

```diff
@@ -33,6 +33,7 @@ def initialize(info = {})
     register_options(
       [
         OptString.new('PASSPHRASE', [ false, 'The configuration password that was set when SecureCRT was installed, if one was supplied']),
+        OptString.new('SESSION_PATH', [ false, 'Specifies the session directory path for SecureCRT']),
```

Comment:
I see this option was added in but you haven't updated the documentation to account for this new option. Could you please go back and add this in?